starknet-replay/src/storage/rpc/class_info.rs

```diff
@@ -0,0 +1,109 @@
+//! This module is responsible for constructing
+//! [`blockifier::execution::contract_class::ClassInfo`] struct which is needed
+//! to convert transactions is the format needed by [`blockifier`] for the
+//! replay.
+
+use blockifier::execution::contract_class::ClassInfo;
+use starknet::core::types::ContractClass;
+use starknet_api::core::ClassHash;
+use starknet_api::transaction::{DeclareTransaction, Transaction};
+
+use super::contract_class::decompress_sierra;
+use crate::block_number::BlockNumber;
+use crate::error::DatabaseError;
+use crate::runner::replay_class_hash::ReplayClassHash;
+use crate::storage::rpc::contract_class::decompress_casm;
+use crate::storage::Storage;
+
+/// This internal function returns the
+/// [`blockifier::execution::contract_class::ClassInfo`] for both Sierra and
+/// CASM contracts.
+///
+/// # Arguments
+///
+/// - `storage`: the struct which implements the trait
+///   [`crate::storage::Storage`] to query the Starknet blockchain.
```

Comment:
Saying the type is unnecessary as the generated doc has the function signature. Same below.